data_structures/src/chain.rs

```diff
@@ -795,6 +822,12 @@ impl fmt::Display for PublicKeyHash {
     }
 }
 
+impl Hashable for PublicKeyHash {
+    fn hash(&self) -> Hash {
+        calculate_sha256(&self.to_pb_bytes().unwrap()).into()
+    }
+}
+
```

Comment:
As discussed with @aesedepece , maybe we could use a blanket implementation in order to avoid code duplication:

```rust
impl<T> Hashable for T
    where T: ProtobufConvert
{
    fn hash(&self) -> Hash {
        calculate_sha256(&self.to_pb_bytes().unwrap()).into()
    }
}
```

Reply:
![image](https://user-images.githubusercontent.com/44392885/81283866-a2729d00-905d-11ea-85e5-da34af9c8625.png)

Reply:
I think it should be do it in a specific issue to fix that, dont you think that?

Reply:
no prob, it was just a suggestion! :)